src/app/layout.tsx

```
@@ -64,6 +94,24 @@ export default function RootLayout( {
     </main>
     {/*</StrictMode>*/}
 
+    <script
+      dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)